airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/mysql/MySqlDebeziumStateUtil.java

```diff
@@ -255,8 +255,8 @@ public JsonNode constructInitialDebeziumState(final Properties properties,
       while (!publisher.hasClosed()) {
         final ChangeEvent<String, String> event = queue.poll(10, TimeUnit.SECONDS);
         if (event == null) {
-          Duration initialWaitingDuration = Duration.ofMinutes(5L);
-          // If initial waiting seconds is configured and it's greater than 5 minutes, use that value instead
+          Duration initialWaitingDuration = Duration.ofMinutes(10L);
+          // If initial waiting seconds is configured and it's greater than 10 minutes, use that value instead
```

Comment:
Just curious but why do we impose a minimum value here? Do our users foot-gun themselves by setting them too low otherwise?

Reply:
Paired with @stephane-airbyte on this. We decided that skipping the timeout and setting the `schema.history.internal.store.only.captured.databases.ddl = true` would reduce the time taken to perform the snapshot